src/components/AllJokes/AllJokesContainer.js

```diff
@@ -8,6 +13,58 @@ import { JokesList } from "./";
 import { Form, Button } from "react-bootstrap";
 
 const AllJoke = () => {
+  const DEFAULT_JOKES_AMMOUNT = 20;
+  const dispatch = useDispatch();
+  const allJokes = useSelector((state) => state.jokes);
+  const apiError = useSelector((state) => state.error);
+  const reduxJokesFilter = useSelector((state) => state.filter);
+  const [jokesFilter, setJokesFilter] = useState(reduxJokesFilter);
+  const [jokesAmmountToFetch, setJokesAmmountToFetch] = useState("");
+
+  function initRequestFetchJokes() {
+    if (!allJokes.length) {
+      dispatch(fetchJokesRequested(DEFAULT_JOKES_AMMOUNT));
+    }
+  }
+
+  useEffect(() => {
+    if(!apiError){
+      initRequestFetchJokes();
+    }
+  }, []);
+
+  useEffect(() => {
+    if (allJokes.length > DEFAULT_JOKES_AMMOUNT) {
+      window.scrollTo(0, document.body.scrollHeight);
+    }
+  }, [allJokes.length]);
+
+  function handleChangeJokesFilter(e) {
```

Comment:
- use es6 syntax everywhere, instead of old javascript
- don't use one symbol variable names. names of the variables should be meaningfull